experiment/build/docker_images.py

```diff
@@ -0,0 +1,56 @@
+# Copyright 2020 Google LLC
```

Comment:
How to deal with `pull-image` jobs?

Reply:
BuildKit (https://github.com/google/fuzzbench/issues/536) should take care of that. 
Also please note that this PR is just a prototype showing how https://github.com/google/fuzzbench/issues/336 can be approached, but @Tanq16 would work on implementing it end to end.

Reply:
Sounds good, thanks.